arch/arm/include/aarch32/cortex_a_r/exc.h

```diff
@@ -38,6 +38,17 @@ static ALWAYS_INLINE bool arch_is_in_isr(void)
 	return (_kernel.cpus[0].nested != 0U);
 }
 
+#if defined(CONFIG_USERSPACE)
+/*
+ * This function is used by privileged code to determine if the thread
+ * associated with the stack frame is in user mode.
+ */
+static ALWAYS_INLINE bool arch_exception_from_user_mode(const z_arch_esf_t *esf)
```

Comment:
`arch_` prefix should be used only by the arch interface functions defined in [`include/sys/arch_interface.h`](https://github.com/zephyrproject-rtos/zephyr/blob/3eb4ea25e51b6a21e3b85ef556933c5ebb56f3c8/include/sys/arch_interface.h).

Since this function is ARM-specific, it should be prefixed `z_arm_`.

Reply:
correct

Reply:
done